client/jetpack-cloud/sections/partner-portal/company-details-form/components/searchable-dropdown.tsx

```diff
@@ -0,0 +1,68 @@
+import { Gridicon } from '@automattic/components';
+import Select, { StylesConfig, DropdownIndicatorProps, components } from 'react-select';
```

Comment:
I'd discourage introducing yet another third-party component for this, given that we have our own alternatives.

Unless I'm missing something, we should be able to use `ComboboxControl` from `@wordpress/components` - see [component docs](https://developer.wordpress.org/block-editor/reference-guides/components/combobox-control/) for more info.

This component is already in use in a couple of places of Calypso:

https://github.com/Automattic/wp-calypso/blob/349a6b6a0fe384efcc8651ce47f0db4f4bf46fa1/client/landing/stepper/declarative-flow/internals/steps-repository/store-address/index.tsx#L246

https://github.com/Automattic/wp-calypso/blob/349a6b6a0fe384efcc8651ce47f0db4f4bf46fa1/client/signup/steps/woocommerce-install/step-store-address/index.tsx#L164

So I don't see a good reason to use another library for the same purpose. If there's a feature missing, we still might prefer to contribute upstream to support that feature, instead of using another 3rd-party library.

Here's some context as to why `@wordpress/components` are preferred:

p3N5Fy-uu-p2

p58i-7Ei-p2

Hope that helps, but please let me know if you run into any trouble or if you have any further questions.

Reply:
@tyxla I tried to use the `ComboboxControl`, but it doesn't seem to support the `disabled` props. In case we want to contribute by adding the missing feature. How does that work on the timing of release? Do we immediately get the changes in Calypso, or will we have to wait for the next Gutenberg release?

Reply:
Ohh I just found out. It looks like there is a weekly release.

Reply:
I also wonder if wrapping the `ComboboxControl` in a `<Disabled />` component will do the job for you.